Meshtastic/MeshtasticAppDelegate.swift

```diff
@@ -27,11 +32,66 @@ class MeshtasticAppDelegate: NSObject, UIApplicationDelegate, UNUserNotification
 		}
 		return true
 	}
+	
+	func addBleManager(bleManager: BLEManager){
+		self.bleManager = bleManager
+	}
 	func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
 	}
 	// This method is called when user clicked on the notification
 	func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
 		let userInfo = response.notification.request.content.userInfo
+		
+		
+		
+		if (bleManager != nil){
```

Comment:
instead of checking for nil and then force unwrapping later, refactor this to use `if let`

example:

```
if let bleManager {
    //...
}
```

This will do the exact same thing, but has safety guarantees baked in, since using `!` can potentially crash in certain situations.